app/helpers/service_helper.rb

```diff
@@ -41,11 +41,18 @@ def service_switch_banner(optional_classes: 'govuk-!-margin-bottom-7')
           tag.p do
             banner_copy
           end
-        end
+        end +
+        steel_safeguards
```

Comment:
Does this show for the XI service?

Reply:
It only works because the `steel_safeguards` function returns an empty string that can be safely concatenated for xi an any page other than the sections, but it works